packages/webdriver-utils/src/metadata/desktopMetaData.js

```diff
@@ -0,0 +1,54 @@
+export default class DesktopMetaData {
+  constructor(driver, opts) {
+    this.driver = driver;
+    this.capabilities = opts;
+  }
+
+  browserName() {
+    return this.capabilities.browserName.toLowerCase();
+  }
+
+  browserVersion() {
+    return this.capabilities.browserVersion.split('.')[0];
+  }
+
+  osName() {
+    let osName = this.capabilities.os;
+    if (osName) return osName.toLowerCase();
+
+    osName = this.capabilities.platform;
+    return osName;
+  }
+
+  // showing major version
+  osVersion() {
+    return this.capabilities.osVersion.toLowerCase();
+  }
+
+  // combination of browserName + browserVersion + osVersion + osName
+  deviceName() {
+    return this.browserName() + '_' + this.browserVersion() + '_' + this.osVersion() + '_' + this.osName();
```

Comment:
Lets change this to:
`this.osName() + '_' + this.osVersion() + '_' + this.browserName() + '_' + this.browserVersion();`